schemas/source.yml

```diff
@@ -28,6 +28,13 @@
     and destination details from a network transaction. If the event also contains identification of the
     client and server roles, then the client and server fields should also be populated.
   type: group
+  reusable:
+    top_level: true
+    expected:
+      - at: process.entry_meta
+        as: source
```

Comment:
perhaps we should add a short_description here? e.g
"Remote client information such as ip, port and geo location."